src/gif/GifSprite.ts

```diff
@@ -22,7 +20,6 @@ import type { SCALE_MODE } from '../rendering/renderers/shared/texture/const';
  *     animationSpeed: 1.5,      // 50% faster than normal
  *     loop: true,               // Loop the animation
  *     autoPlay: true,           // Start playing immediately
- *     scaleMode: 'nearest',     // Pixel art style scaling
```

Comment:
It's a breaking change (minor, yet breaking) for strictly typed projects. 
Maybe we can introduce a deprecation warning and update the documentation, explaining how to apply nearest scale mode on load?

Reply:
For example, we can add an example to the top level of the class documentation.

Reply:
Good feedback. Added two examples and a deprecation for scaleMode in options.